cmd/cloud/deployment.go

```diff
@@ -360,3 +371,50 @@ func deploymentVariableUpdate(cmd *cobra.Command, args []string, out io.Writer)
 
 	return deployment.VariableModify(deploymentID, variableKey, variableValue, ws, envFile, variableList, useEnvFile, makeSecret, true, astroClient, out)
 }
+
+func newDeploymentWorkerQueueRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "worker-queue",
+		Aliases: []string{"wq"},
+		Short:   "Manage deployment worker queues",
+		Long:    "Manage worker queues for an Astro Deployment.",
+	}
+	cmd.AddCommand(
+		newDeploymentWorkerQueueCreateCmd(out),
+	)
+	return cmd
+}
+
+func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create a Deployment's worker queue",
+		Long:    "Create a worker queue for an Astro Deployment",
+		Example: "", // TODO should we add some examples for default and custom worker-queue commands?
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deploymentWorkerQueueCreate(cmd, args, out)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "The deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&wQueueName, "name", "n", "", "The name of the worker queue. If the name contains a space, specify the entire name within quotes \"\" .")
+	cmd.Flags().BoolVarP(&isDefaultWQueue, "isDefault", "", false, "Make this the default worker queue by setting this to true.")
+	cmd.Flags().IntVarP(&wQueueMin, "min-count", "", 0, "The min worker count of the worker queue. Possible values are between 1 and 10.")
+	cmd.Flags().IntVarP(&wQueueMax, "max-count", "", 0, "The max worker count of the worker queue. Possible values are between 11 and 20.")
```

Comment:
Same comment here if the max default is 0, no pods will ever spin up. The max default set in the UI / Astrohub is 10 for reference.

Reply:
Also `11 and 20` - are these placeholders? They don't reflect the numbers we impose in the UI

Reply:
> Also `11 and 20` - are these placeholders? They don't reflect the numbers we impose in the UI

@frank-ye Yes these are placeholders as I was trying to not hard code astrohub's defaults in CLIs help messages. Thinking of how to do this so we do not have to change the CLI's messages each time astrohub defaults change. How often do we change these defaults in astrohub?

Reply:
These defaults don't change often, there's also a query that the UI invokes to fetch these default values. It could be overkill though.